src/components/monitor/monitor.jsx

```diff
@@ -21,44 +23,62 @@ const categories = {
 
 const modes = {
     default: DefaultMonitor,
-    large: LargeMonitor
+    large: LargeMonitor,
+    slider: SliderMonitor,
+    list: ListMonitor
 };
 
 const MonitorComponent = props => (
     <ContextMenuTrigger id={`monitor-${props.label}`}>
         <Draggable
             bounds=".monitor-overlay" // Class for monitor container
+            cancel=".no-drag" // Class used for slider input to prevent drag
             defaultClassNameDragging={styles.dragging}
             onStop={props.onDragEnd}
         >
             <Box
                 className={styles.monitorContainer}
                 componentRef={props.componentRef}
-                onDoubleClick={props.onNextMode}
+                onDoubleClick={props.mode === 'list' ? null : props.onNextMode}
             >
                 {(modes[props.mode] || modes.default)({ // Use default until other modes arrive
                     categoryColor: categories[props.category],
                     label: props.label,
-                    value: props.value
+                    value: props.value,
+                    width: props.width,
+                    height: props.height,
+                    min: props.min,
+                    max: props.max
                 })}
             </Box>
         </Draggable>
-        <ContextMenu id={`monitor-${props.label}`}>
-            <MenuItem onClick={props.onSetModeToDefault}>
-                <FormattedMessage
-                    defaultMessage="normal readout"
-                    description="Menu item to switch to the default monitor"
-                    id="gui.monitor.contextMenu.default"
-                />
-            </MenuItem>
-            <MenuItem onClick={props.onSetModeToLarge}>
-                <FormattedMessage
-                    defaultMessage="large readout"
-                    description="Menu item to switch to the large monitor"
-                    id="gui.monitor.contextMenu.large"
-                />
-            </MenuItem>
-        </ContextMenu>
+        {props.mode === 'list' ? null : (
```

Comment:
This is kind of doing work we already have figured out in the container, could we just pass the filtered list of available modes for this instead?